views/term_table.erb

```diff
@@ -107,130 +99,146 @@
     <div class="page-section page-section--grey">
         <div class="container">
 
-            <div class="term-membership-table__title">
-                <h2>Members</h2>
+            <h2 class="text-center">
+                Members
+                <a href="../../download.html" class="download-members" data-ga-track-click="Download term data">
+                    <i class="fa fa-download"></i> Download data
+                </a>
+            </h2>
 
-                <span class="person-card-filter">
-                    <label for="filter-input" class="fa fa-search"></label>
-                    <input type="text" class="js-filter-input" id="filter-input" placeholder="Search by name, place, etc…" data-ga-track-change="Filter field">
-                </span>
+            <div class="js-fixed-parent">
 
-                <div class="download-options">
-                  <!-- download link as button? -->
-                  <a class="button button--quarternary" href="../../download.html" data-ga-track-click="Download term data">
-                    <i class="fa fa-download"></i>
-                    Download <span class="large-screen-only">data</span>
-                  </a>
+                <div class="js-fixed-child">
+                    <div class="card-filters">
+                        <p class="card-filters__filter">
+                            <label for="filter-input">Search for:</label>
+                            <span>
+                                <input type="text" class="form-control js-filter-input" id="filter-input" placeholder="Name, place, etc…" data-ga-track-change="Filter field">
+                            </span>
+                        </p>
+                        <p class="card-filters__filter">
+                            <label for="show-facet">Show details:</label>
+                            <span>
+                                <select class="js-show-facet form-control" id="show-facet">
+                                    <option value="bio">Biographical</option>
+                                    <option value="social">Social links</option>
+                                    <option value="contacts">Contact details</option>
+                                    <option value="identifiers">Identifiers</option>
+                                </select>
+                            </span>
+                        </p>
+                    </div>
                 </div>
-            </div>
 
-            <div class="grid-list" data-active-section="bio">
```

Comment:
Sorry it appears I've changed lots of lines in this file – in actuality, all I did was indent it all 4 spaces further, so I could wrap it in the `<div class="js-fixed-parent">`.